test/integration/spec/localparticipant.js

```diff
@@ -802,11 +827,11 @@ describe('LocalParticipant', function() {
       };
     });
 
-    after(() => {
+    after(async () => {
       thisAudioTrack.stop();
       thisVideoTrack.stop();
-      thisRoom.disconnect();
-      thatRoom.disconnect();
+      [thisRoom, thatRoom].filter(room => room).forEach(room => room.disconnect());
```

Comment:
Do we need to filter?

Reply:
@syerrapragada If a test fails, the Room objects will be `undefined`. So it is just safe to check for non-undefined Room objects.